src/systems/eu/implementations/vanilla/packages/eu-component-link/eu-component-link.scss

```diff
@@ -69,6 +69,34 @@
     }
     /* stylelint-enable */
   }
+
+  .ecl-link--cta {
+    background-color: $ecl-color-yellow-100;
+    box-sizing: border-box;
+    color: $ecl-color-black-100;
+    display: inline-block;
+    font: $ecl-font-m;
+    font-weight: $ecl-font-weight-bold;
+    padding: $ecl-spacing-s $ecl-spacing-m;
+    text-decoration: underline;
```

Comment:
this is already defined in ecl-link, no need to duplicate (same for hover and outline in focus)